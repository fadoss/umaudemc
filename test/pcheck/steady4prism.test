# Steady-state probabilities specifically for Prism
#+ for: prism

# The Jacobi method that PRISM uses by default does not converge in the
# following cases, so we indicate -gs to use Gauss-Seidel instead.

umaudemc pcheck coin head @steady --assign metadata -- -gs
umaudemc pcheck coin head @steady --assign 'uaction(ttail=3, thead=7)' -- -gs
umaudemc pcheck coin head @steady --assign @ite.assign -- -gs
umaudemc pcheck coin head @steady 'choice(3 : ttail, 7 : thead) *' --assign strategy -- -gs
