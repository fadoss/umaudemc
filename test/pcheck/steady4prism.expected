 0.6153846153846154   head
 0.3846153846153847   tail
 0.7000000000000001   head
 0.3                  tail
 0.7                  head
 0.3                  tail
 0.7                  head
 0.3                  tail
